database/migrations/default/1746807631787_patient_inj_sev_update/up.sql

```diff
@@ -0,0 +1,258 @@
+alter table ems__incidents drop column patient_injry_sev_id;
+
+alter table ems__incidents
+add column patient_injry_sev_id integer generated always as (
+    case
+        --
+        -- fatal injuries
+        --
+        when
+            (lower(pcr_provider_impression_primary) = 'death on scene')
+            or
+            (lower(pcr_provider_impression_secondary) = 'death on scene')
+            or
+            (lower(pcr_outcome) = 'deceased on scene')
+            or
+            (
+                lower(pcr_patient_acuity_final)
+                = 'dead without resuscitation efforts (black)'
+            )
+            or (
+                (
+                    lower(pcr_patient_acuity_initial)
+                    = 'dead without resuscitation efforts (black)'
+                )
+                and (
+                    pcr_patient_acuity_final is NULL
+                )
+            )
+            then 4
+        --
+        -- serious injuries
+        --
+        when
+            (lower(pcr_patient_acuity_final) = 'critical (red)')
+            or
+            (
+                lower(pcr_patient_acuity_initial) = 'critical (red)'
+                and pcr_patient_acuity_final is NULL
+            )
+            or
+            (
+                lower(pcr_patient_acuity_final) = 'emergent (yellow)'
+                and lower(pcr_outcome) in ('transported', 'care transferred')
+            )
+            or
+            (
+                lower(pcr_patient_acuity_initial) = 'emergent (yellow)'
+                and lower(pcr_patient_acuity_final) is NULL
+                and lower(pcr_outcome) in ('transported', 'care transferred')
+            )
+            or
+            (right(pcr_transport_priority, 1) = '3')
+            or
+            (lower(pcr_transport_priority) like 'charlie%')
+            or
+            (lower(pcr_transport_priority) like 'delta%')
+            or
+            (lower(pcr_transport_priority) like 'echo%')
```

Comment:
The `pcr_transport_priority` column uses a mix of letter codenames plus generic `Code 1` and `Code 3. Our current best understanding:

- Code 1: transport without lights + sirens
- Code 3: transport with lights + sirens

And for the latter codes:

- A (Alpha Response) refers to a low-priority situation.
- B (Bravo Response) indicates a mid-priority condition.
- C (Charlie Response) represents a potentially life-threatening condition.
- D (Delta Response) indicates a life-threatening emergency.
- E (Echo Response) refers to full cardiac arrest or imminent death.